docs/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@babel/helper-get-function-arity": "7.16.7",
+    "@cookbookdev/docsbot": "*",
```

Comment:
Please use a fixed version here! It's important to avoid failing when the dependency changes